includes/templates/admin/dashboard.php

```diff
@@ -31,10 +31,10 @@
 
 $dashboard_settings = \Google\Web_Stories\Services::get( 'dashboard' )->get_dashboard_settings();
 
-$init_script = <<<JS
-	wp.domReady( function() {
-	  webStories.initializeStoryDashboard( 'web-stories-dashboard', %s );
-	} );
+$init_script = <<<'JS'
+    wp.domReady( function() {
+      webStories.initializeStoryDashboard( 'web-stories-dashboard', %s );
+    } );
```

Comment:
Is this change in indentation intentional?

Reply:
https://github.com/GoogleForCreators/web-stories-wp/pull/12672#discussion_r1026294306